.travis.yml

```diff
@@ -1,9 +1,8 @@
 language: node_js
 node_js:
-  - "0.10"
-  - "0.12"
```

Comment:
I think these versions of node should still be supported at least through the end of 2016 since they are still valid LTS versions.